Libraries/PushNotificationIOS/RCTPushNotificationPlugins.h

```diff
@@ -9,7 +9,8 @@
 
 #ifdef RN_DISABLE_OSS_PLUGIN_HEADER
 
-// FB Internal: FBRCTPushNotificationPlugins.h is autogenerated by the build system.
```

Comment:
All the changes outside the `.github` folder should be amended as they're not relevant.

Reply:
yeah, sorry about that - these files were changed when I ran the linter, and I accidentally included them without noticing. Will revert changes to those